Cypress Accessibility and MCP docs updates (#6412)
cypress-io/cypress-documentation: docs/accessibility/core-concepts/element-identification.mdx

```

Refer to the [Significant Attributes](/accessibility/configuration/significantattributes) guide for detailed instructions.

### Component attributes

Require extra attributes (such as a component or team identifier) in the **violation target selectors** shown in reports, to provide consistency and readability in your selectors:

```json
{
  "accessibility": {
    "components": {
      "componentAttributes": [
        { "attributeName": "data-component-name", "includeInSelector": true }
      ]
    }
  }
}
```

See the [components](/accessibility/configuration/components) configuration guide for `componentAttributeFilters`, defaults, and examples.

## Differences compared to Axe-Core® element identification

The Cypress Cloud process replaces the underlying Axe-Core® library's default mechanism for identifying elements.

> Axe-Core® captures a unique identifier for a given element in a specific DOM context, focused on one single state. This default behavior is not intended to account for situations where sometimes tens of thousands of Axe-Core® scans need to be merged together for a single run, or later compared based on different builds of the application.

Reviewer: ```suggestion
Axe-Core® captures a unique identifier for a given element in a specific DOM context, focused on one single state. This default behavior does not account for merging thousands of Axe-Core® scans into a single report or for comparing scans of the application as the application changes.
```
